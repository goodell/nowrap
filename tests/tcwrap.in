ThisIsAVeryLongText
ThatShouldBeBrokenIntoMultipleLinesAndWrappedAccordingly.
AlsoWorks	With	TabCharacter	A	ny		where, also	mixed with 日本	語 inside 日本語.
