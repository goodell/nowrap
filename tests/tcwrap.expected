ThisIsAVer
yLongText
ThatShould
BeBrokenIn
toMultiple
LinesAndWr
appedAccor
dingly.
AlsoWorks
	Wi
th	Ta
bCharacter
	A
	ny
	
	wh
ere, also
	mi
xed with 
日本	語
 inside 日
本語.
